### DISCUSSION TITLE
How do I decrease the "activate menu" tap zones height?

### DISCUSSION THREAD

**Discussion by wa15** [koreader/koreader#7781] [Q&A] [answered]
I have my activate menu setting as only tap, and I would like to decrease the height only of the tap zones as shown in my pic(from gray to red). Which specific value in advanced settings would i need to change? (Im on android, and the thing i actually wanted was to disable fullscreen and show the notification bar, but it doesnt seem i can do this) thanks for the help
![Screenshot_20210530-190510_KOReader](https://user-images.githubusercontent.com/57420708/120123118-05f4ce80-c17b-11eb-8613-b1e8fae4dfd4.jpg)

**Comment by NiLuJe** [comment, marked as answer]
That would be these four: https://github.com/koreader/koreader/blob/d5a2df3c0447e94d02d10b98b17f04b9351b81fc/defaults.lua#L78-L81

(You can safely set the `_EXT` ones to an empty rectangle if you just want a single, simple rectangle, as per your screenshot).

Keep in mind that you'd be awfully close to the footer tapzone for config there, and the footer has priority.

**Reply by NiLuJe** [reply]
Visual aid (in blue): https://github.com/koreader/koreader/pull/6918#issuecomment-736856442

**Reply by wa15** [reply]
So my setup would be 1) a single, simple thin rectangle to tap for both menus and 2) completely disable the bottom menu tap zone. 
I have auto-show bottom menu enabled and gestures plugin disabled. I zeroed out everything except for 2. Looking at these changes, did I do anything unnecessary?
DTAP_ZONE_MENU = {x = 0, y = 0, w = 1, h = .05} 
DTAP_ZONE_MENU_EXT = {x = 0, y = 0, w = 0, h = 0}  
DTAP_ZONE_CONFIG = {x = 0, y = 0, w = 0, h = 0} 
DTAP_ZONE_CONFIG_EXT = {x = 0, y = 0, w = 0, h = 0}

**Reply by NiLuJe** [reply]
Yep, that would do it.

(That might be *really* thin, but you know what to tweak ;p)

**Reply by wa15** [reply]
Thank you!
